### PR TITLE
[ACT-60] Spaces & Channels product pages

### PR DESCRIPTION
**Pull request by denissellu**
## Description

This is just the design for the Spaces and channels, Copy not yet added

* [ACT-60](https://ably.atlassian.net/browse/ACT-60) - Spaces product page https://ably-docs-act-60-spaces-uftjkw.herokuapp.com/docs/products/spaces
* [ACT-61](https://ably.atlassian.net/browse/ACT-61) - Channels product page https://ably-docs-act-60-spaces-uftjkw.herokuapp.com/docs/products/channels

## Review

Instructions on how to review the PR. 

* [Page to review](link)

[ACT-60]: https://ably.atlassian.net/browse/ACT-60?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ACT-61]: https://ably.atlassian.net/browse/ACT-61?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/ProductPage/BodySection/BodySection.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import { SectionProps } from '../HomepageContent';
+import { BodySectionDescription } from './BodySectionDescription';
+import { FeatureCard } from './Card/FeatureCard';
+
+const cardTypes = {
+  feature: FeatureCard,
+};
+
+const betaPillStyle = {
+  fontSize: '14px',
```
Comment (kennethkalmer) [inline src/components/ProductPage/BodySection/BodySection.tsx:18]: I think for these we should use CSS variables where we can, like:

~~~js
{
  fontSize: 'var(--fs-p3)', // should be 14px
}
~~~

Line height of `1.25rem` can be done with the `leading-normal` class from tailwind.

File: src/components/ProductPage/BodySection/BodySection.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <h1 className={`ui-text-${section.level} mt-40 col-span-6 flex items-center`}>
+          {section.title}
+          <span style={betaPillStyle} className="px-12 py-4 ml-12">
+            {section.releaseStage}
+          </span>
+        </h1>
+      ) : section.level ? (
+        <section.level className={`ui-text-${section.level} mt-40 col-span-6`}>{section.title}</section.level>
+      ) : (
+        <h2 className="ui-text-h2 col-span-6">{section.title}</h2>
```
Comment (kennethkalmer) [inline src/components/ProductPage/BodySection/BodySection.tsx:49]: ✂️ `col-span-6`

File: src/components/ProductPage/ProductPageContent.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  level: keyof JSX.IntrinsicElements | null;
+  description: string | null;
+  columns: number;
+  releaseStage: string;
+  callToAction: CallToActionProps;
+  cards: CardProps[];
+};
+
+export const ProductPageContent = ({ sections }: { sections: SectionProps[] }) => (
+  <article className="px-24 md:pl-40 md:pr-48 xl:pr-64 col-span-2 lg:col-start-2 2xl:col-start-1">
```
[the remark this answers is not in the document]
Comment (denissellu) [reply src/components/ProductPage/ProductPageContent.tsx:36]: ✂️

**Review by kennethkalmer — APPROVED** [review]

**Pull request merged**